### PR TITLE
Add breadcrumb and fixing bugs

### PR DESCRIPTION
**Pull request by XiaoxuHU**
1. Adding breadcrumb for navigation from parent detector to child

image

2. Adding failed loading section for child detectors loading not successful
image

3. Bug Fixing
    3.1 Read startTime and endTime from url
    3.2 Focus will reset to time-picker when detector loading is finished
    3.3 Three loading dots in parent-child detector will disappeared after loading is done
    3.4 Metric below highchart will only show data without prefix points
    3.5 Revert to use stack chart as default

### CODE REVIEW

File: AngularApp/projects/diagnostic-data/src/lib/components/detector-list/detector-list.component.html [hunk 1]
```diff
@@ -110,6 +114,39 @@
     </collapsible-list-item>
   </collapsible-list-fabric>
 </div>
+<br />
+
+<div *ngIf="failedLoadingViewModels.length > 0">
+  <collapsible-list-fabric [collapsed]="false" [title]="'Failed Loading Checks (' + failedLoadingViewModels.length + ')'">
```
Comment (ShekharGupta1988) [inline AngularApp/projects/diagnostic-data/src/lib/components/detector-list/detector-list.component.html:120]: nitpick : just get a review of this tittle from Yun Jung and Hanna once and see if they have any feedback.

**Review by ShekharGupta1988 — APPROVED** [review]

File: AngularApp/projects/applens/src/app/modules/dashboard/tabs/tab-common/tab-common.component.ts [hunk 2]
```diff
@@ -41,22 +41,26 @@ export class TabCommonComponent implements OnInit {
     switch (key) {
       case TabKey.Data:
         this._router.navigate(["./"], {
-          relativeTo: this._activatedRoute
+          relativeTo: this._activatedRoute,
```
Comment (cindylovescoding) [inline AngularApp/projects/applens/src/app/modules/dashboard/tabs/tab-common/tab-common.component.ts:44]: Why do we need to add this?

**Review by cindylovescoding — APPROVED** [review]

File: AngularApp/projects/diagnostic-data/src/lib/components/detector-list/detector-list.component.html (hunk 1 above)
Comment (XiaoxuHU) [reply AngularApp/projects/diagnostic-data/src/lib/components/detector-list/detector-list.component.html:120]: Yes, Shekhar !  I checked Hanna about the wording

File: AngularApp/projects/applens/src/app/modules/dashboard/tabs/tab-common/tab-common.component.ts (hunk 2 above)
Comment (XiaoxuHU) [reply AngularApp/projects/applens/src/app/modules/dashboard/tabs/tab-common/tab-common.component.ts:44]: Bug reported by Nazeef, if not add queryParmasHandling, some query params will missing when switching between the tabs

**Pull request merged**